libirods_rule_engine_plugin-indexing.cpp

```diff
@@ -936,7 +936,20 @@ irods::error exec_rule_text(
                 _e.code(),
                 _e.what());
     }
-
+    catch(const std::exception & _e) {
+        rodsLog(LOG_ERROR,"std::exception (%s) in FILE %s LINE %d FUNCTION %s ",
+                            _e.what(),__FILE__,__LINE__,__FUNCTION__);
+        return ERROR(
+                SYS_NOT_SUPPORTED,
```

Comment:
Can we make this `SYS_INTERNAL_ERR`? This is the convention I had been following when working on other things. If you feel this error code is more appropriate in this situation, then let's stick with that. :)